### DISCUSSION TITLE
Update on the development branch

### DISCUSSION THREAD

**Discussion by kaiyux** [NVIDIA/TensorRT-LLM#350] [Announcements]
Hi,

The TensorRT-LLM team is pleased to announce that we are pushing an update to the [development](https://github.com/NVIDIA/TensorRT-LLM/tree/main) branch(also includes the [Triton backend](https://github.com/triton-inference-server/tensorrtllm_backend/tree/main)) this November 10th, 2023.

This update includes:

* Model Support
  - Mistral 7B
* Features
  - Python binding for C++ runtime.
  - Support for stop and bad words in non-inflight batching.
  - New heuristic for launching Multi-block MMHA.
* Bug fixes
  - Fix missing torch dependencies: https://github.com/NVIDIA/TensorRT-LLM/issues/151, https://github.com/NVIDIA/TensorRT-LLM/issues/277
  - Fix logits dtype for Baichuan and ChatGLM: https://github.com/NVIDIA/TensorRT-LLM/issues/335

Thanks,
The TensorRT-LLM Engineering Team
